Amazon_Interview_Questions/Medium/Find_Peak_Element.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+
+Problem Link: https://leetcode.com/problems/find-peak-element/
+
+*/
+
+class Solution {
+public:
+    int findPeakElement(vector<int>& nums) {
+        int lo = 0, hi = nums.size()-1, mid;
+        
+        while (lo < hi) {
+            mid = lo + (hi - lo) / 2;
```

Comment:
good! naming of variables matters a lot during interviews.

Reply:
@SaumyaSingh1, Thank you for the appreciation mam. The idea is good but I am working on a project and learning DP and Graphs, I hardly solve questions from these two topics and still learning,
Given two coding rounds of companies, Result: couldn't clear. They asked from DP and Graph. 
**Can you give some suggestions regarding the same? How to study DP and Graph.**
And placement season is nearby, I am looking for an SWE/SDE internship, 
Since I am solving these leetcode questions and was unable to clear out the coding round, found myself losing confidence. 
**Are these questions sufficient for cracking coding rounds and interviews?**

Reply:
Don't lose confidence, Deepak. Rejections will be many but learn from them, practice even better and I am sure you will clear them. 
I practiced DP and Graphs from leetcode and youtube only. Don't be scared of those topics. 
It's just the beginning. 
@dpkk1

Reply:
I will try my best mam, thank you :)